csharp/codeql-extractor.yml

```diff
@@ -16,3 +16,15 @@ file_types:
     extensions:
       - .cs
 legacy_qltest_extraction: true
+options:
+  buildless:
+    title: Whether to use buildless (standalone) extraction.
+    description: >
+      A value indicating, which type of extraction the autobuilder should perform.
+      If 'true', then the standalone extractor will be used, otherwise tracing extraction
+      will be performed.
+      The default is 'false'.
+      Note that buildless extraction will generally yield less accurate analysis results,
+      and should only be used in cases where it is not possible to build
+      the code (for example if it uses inaccessible dependencies).
+    type: string
```

Comment:
I think we should add `pattern: "^(false|true)$"`

Reply:
Yes, that is a good idea - will add that.